### DISCUSSION TITLE
EEG-Notebooks Roadmap

### DISCUSSION THREAD

**Discussion by JohnGriffiths** [NeuroTechX/EEG-ExPy#64] [Ideas]
@oreHGA correctly pointed out just now at the NTX hacknight that it would be useful to have a roadmap document. 

This would spell out the short, medium, and long term objectives of the project, and would be helpful for new (and old!) contributors to understand where they might most usefully focus their efforts.

@oreHGA - could you share any further thoughts on what this should include, and share some links of good examples of this kind of thing from other projects?

@jnaulty @mhough

**Comment by oreHGA** [comment]
Sure thing @JohnGriffiths! [Github's Public Roadmap](https://github.com/github/roadmap) has a good introduction & a "guide to using the roadmap" sections that we can leverage.

Adding [issue templates](https://github.com/microsoft/PowerToys/tree/master/.github/ISSUE_TEMPLATE) in the project `.github` repo would also make it easier for people filing new issues & maintainers having enough information to answer requests.


(I think there are already some open issues related to these but) Some objectives mentioned were:
- Codebase hygiene
- Documentation updates
- Expanding example dataset for experiments on different devices
- Adding code for more experiments
- Exploring ways to reduce setup time
